utils/utils.go

```diff
@@ -38,25 +39,37 @@ import (
 type ContainerRuntime string
 
 const (
-	DOCKER ContainerRuntime = "docker"
-	PODMAN ContainerRuntime = "podman"
+	DOCKER     ContainerRuntime = "docker"
+	PODMAN     ContainerRuntime = "podman"
+	CONTAINERD ContainerRuntime = "containerd"
+
+	MACOS = runtime.GOOS == "darwin"
 
 	marinerImageVariantName = "mariner"
 
 	socketFormat = "%s/dapr-%s-%s.socket"
 )
 
+var IsValidRuntimeOnMacos = func(containerRuntime string) bool {
+	containerRuntime = strings.TrimSpace(containerRuntime)
+	return containerRuntime == string(CONTAINERD) && !MACOS
+}
+
 // IsValidContainerRuntime checks if the input is a valid container runtime.
-// Valid container runtimes are docker and podman.
+// Valid container runtimes are docker and podman and containerd.
 func IsValidContainerRuntime(containerRuntime string) bool {
 	containerRuntime = strings.TrimSpace(containerRuntime)
-	return containerRuntime == string(DOCKER) || containerRuntime == string(PODMAN)
+	return containerRuntime == string(DOCKER) || containerRuntime == string(PODMAN) || IsValidRuntimeOnMacos(containerRuntime)
 }
 
 // GetContainerRuntimeCmd returns a valid container runtime to be used by CLI operations.
-// If the input is a valid container runtime, it is returned as is.
+// If the input is a valid container runtime, it is returned client tool.
 // Otherwise the default container runtime, docker, is returned.
 func GetContainerRuntimeCmd(containerRuntime string) string {
+	// containerd runtime use nerdctl tool.
+	if IsValidRuntimeOnMacos(containerRuntime) {
+		return "nerdctl"
```

Comment:
putting this also as constant would be better ?

Reply:
👍